LayoutTests/model-element/model-element-width-height.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html><!-- webkit-test-runner [ ModelElementEnabled=true ] -->
+<script src="../resources/testharness.js"></script>
+<script src="../resources/testharnessreport.js"></script>
+<body>
+<script>
+test(() => {
+    let model = document.createElement("model");
```

Comment:
Don't shy away from using `const` :) Or are you one of those C++ people who don't want to use it because it's not a _real_ `const`?

Reply:
I am somewhat anti-const because it's so much longer than "let"!